app/views/admin/kaminari/_next_page.html.haml

```diff
@@ -1,2 +1,6 @@
-%li.list-inline-item
+%li.list-inline-item.p-2.d-none.d-md-inline
```

Comment:
Rather than managing this via two different list items that are shown or hidden, you can do something like this:

```haml
= link_to_unless current_page.last?, url, :rel => 'next', :remote => remote, class: "btn btn-secondary", role: "button" do
  = "&rsaquo;"
  %span.d-none.d-md-inline
    = raw(t 'views.pagination.next')
```